governance-exercises/Le Bateleur - Interpretation Exercise 01/GOV 12 Rewrite.md

```diff
@@ -0,0 +1,56 @@
+
+Governance Facilitators may initiate a GOV 12 Governance Poll only if *all* of the following conditions are met:
+
+- The Governance Facilitator must signal their intention to initiate a GOV 12 Governance Poll in a Forum post. The Forum post must document, in reasonable detail, the Facilitator's 1) analysis and reasoning in undertaking the balancing inquiry and 2) the proposed revisions to the Atlas/Scope Artifacts. The post must contain a formal conflict-of-interest declaration. 
+- Two endorsements of the Governance Facilitator's post are required. One Facilitator *and* one Advisory Council Member of the impacted Scope(s) must post on the Forum their explicit endorsement. The endorsers must substantively explain their support of the Governance Facilitator's proposal. The endorsers must also include a formal conflict-of-interest declaration. 
+- In order to qualify as a GOV 12 endorser, the Facilitator and Advisory Council Member must be currently active in their respective roles and otherwise be in good standing. If a Facilitator has received a Formal Warning per [GOV 4.2](https://mips.makerdao.com/mips/details/MIP113#4-2-alignment-conserver-formal-warnings), they cannot qualify as a GOV 12 endorser.
+- Once a Facilitator/Advisory Council Member has acted in the capacity of a GOV 12 endorser, they are prohibited from acting in such a capacity again until two full calendar months have passed. 
+
+#### 12.4 Exceptions to the General Protocol
+
+##### 12.4.1 Clear Mistakes
+
+Excepted from the General Protocol are *clear* mistakes, such as typos or inconsistencies, in the text of the Atlas/Scope Artifacts that obstruct the critical functioning of Maker Governance. 
+
+'Clear' typos/inconsistencies are defined as those for which substantive supporting documentation evincing the drafter's intent can be cited, and concerning whose interpretation, no reasonable disagreement can arise. Because of the nature of clear typos/inconsistencies, they can typically be corrected with a minimal amount of revision. 
+
+In such a scenario, the Governance Facilitators must signal their intention via a Forum Post that briefly describes the clear typo/inconsistency, the impact on a critical function of Maker Governance and the proposed revision to the Atlas/Scope Artifacts. After the Forum post, the Facilitators may directly proceed to initiating the Governance Poll.   
+
+##### 12.4.2 Launch Project Needs
+
+Excepted from the General Protocol are Launch Project-related [needs](https://mips.makerdao.com/mips/details/MIP108#9-launch-project) deemed time-sensitive and high priority by Maker Core. In such a scenario, the Governance Facilitators must signal their intention via a Forum Post that briefly describes the Launch Project-related need (in such a way as to preserve the needed confidentiality) and the proposed revision to the Atlas/Scope Artifacts. After the Forum post, the Facilitators may directly proceed to initiating the Governance Poll. 
```

Comment:
Maker Core doesn't really have opinions, better to term it Maker governance, imo. Both of these are just sweeping the problem under the rug though. How is it determined whether governance/core considers something time-sensitive or high-priority?